ZeldaFullEditor/ZSCustomOverworld.asm

```diff
@@ -9,68 +48,256 @@ pushpc
+
+; W8
+; Enable/Disable subscreen.
+; $00DA63
+!Func00DA63 = $01
+
+; T2.5 (probably)
+; Makes the game decompress the 3 static OW tile sheets on transition.
+; $00D585
+!Func00D585 = $01
+
+; Changes the InitTilesets function to call from the long tables.
+; $006221
+!Func00E221 = $01
+
+; Zeros out the BG color when mirror warping to the pyramid area.
+; $00EEBB
+!Func00EEBB = $01
+
+; W9 BG scrolling for HC and the pyramid area.
+; $00FF7C
+!Func00FF7C = $01
+
+; E2
+; Changes the function that loads overworld properties when exiting a dungeon.
+; Includes removing asm that plays music in certain areas and changing how
+; animated tiles are loaded.
+; $0283EE
+!Func0283EE = $01
+
+; Changes a function that loads animated tiles under certain conditions.
+; $028632
+!Func028632 = $01
+
+; E1
+; Changes part of a function that changes the sub mask color when leaving
+; dungeons.
+; $029AA6
+!Func029AA6 = $01
+
+; T2 S2 W2
+; Main subscreen loading function.
+; $02AF58
+!Func02AF58 = $01
+
+; W1
+; turns on subscreen for pyramid.
+; $02B2D4
+!Func02B2D4 = $01
+
+; W6
+; Activate subscreen durring pyramid warp.
+; $02B3A1
+!Func02B3A1 = $01 
+
+; Controls overworld vertical subscreen movement for the pyramid BG.
+; $02BC44
+!Func02BC44 = $01
+
+; T4 Changes how the pyramid BG scrolls durring transition.
+; $02C02D
+!Func02C02D = $01
+
+; W3 Main palette loading routine.
+; $02C692
+!Func02C692 = $01 
+
+; Rain animation code.
+; $02A4CD
+!Func02A4CD = $01
+
+; T1
+; Mosaic.
+; $02AADB
+!Func02AADB = $01
+
+; T3
+; Transition animated and main palette.
+; $02ABBE
+!Func02ABBE = $01
+
+; Loads the animated tiles after the overworld map is closed.
+; $0ABC5A
+!Func0ABC5A = $01
+
+; Loads different animated tiles when returning from bird travel.
+; $0AB8F5
+!Func0AB8F5 = $01
+
+; W5
+; Load overlay, fixed color, and BG color.
+; $0BFEC6
+!Func0BFEC6 = $01
+
+; S1 W4
+; Transparent color durring warp and during special area enter.
+; $0ED627
+!Func0ED627 = $01
+
+; Resets the area special color after the screen flashes.
+; $0ED8AE
+!Func0ED8AE = $01
+
+; If 1, all of the default vanilla pool values will be applied. 00 by default.
+!UseVanillaPool = $00
+
+; Use this var to disable all of the debug vars above.
+!AllOff = $00
+
+if !AllOff == 1
+!Func00D8D5 = $00
+!Func00DA63 = $00
+!Func00D585 = $00
+!Func00E221 = $00
+!Func00EEBB = $00
+!Func00FF7C = $00
+!Func0283EE = $00
+!Func028632 = $00
+!Func029AA6 = $00
+!Func02AF58 = $00
+!Func02B2D4 = $00
+!Func02B3A1 = $00 
+!Func02BC44 = $00
+!Func02C02D = $00
+!Func02C692 = $00 
+!Func02A4CD = $00
+!Func02AADB = $00
+!Func02ABBE = $00
+!Func0ABC5A = $00
+!Func0AB8F5 = $00
+!Func0BFEC6 = $00
+!Func0ED627 = $00
+!Func0ED8AE = $00
+endif
 
 ; ==============================================================================
 ; Fixing old hooks:
 ; ==============================================================================
 
+; TODO: Eventually remove these? I'm not sure. If anyone uses an old ZS on their
```

Comment:
Might be good for this to be included as some sort of separate ZS utils file and maybe use asar functions/directives to make permanent fixes to a ROM based on legacy edits/hooks.